### ISSUE TITLE
ARM Template not available

### ISSUE THREAD

**Issue by ShinraCorpMixer** [MicrosoftDocs/azure-gaming-docs#4]
I've recieved an error after clicking on "Deploy to Azure" The error suggests that the ARM template isn't available.

"There was an error loading the template from URI 'https://raw.githubusercontent.com/dgkanatsios/AzureContainerInstancesManagement/master/deploy.json'. Ensure that the template is publicly accessible and does not have any comments."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 697e5b68-adbf-e20d-cb28-a2651891c82f
* Version Independent ID: a5beac2b-eedc-e696-f38b-59e43f059504
* Content: [Multiplayer Game Server Hosting Using ACI - Azure Gaming](https://docs.microsoft.com/en-us/gaming/azure/reference-architectures/multiplayer-synchronous-aci)
* Content Source: [azure-gaming-pr/reference-architectures/multiplayer-synchronous-aci.md](https://github.com/MicrosoftDocs/azure-gaming-docs/blob/docs/azure-gaming-pr/reference-architectures/multiplayer-synchronous-aci.md)
* Product: **azure-gaming**
* GitHub Login: @BrianPeek
* Microsoft Alias: **brpeek**

**Comment by BrianPeek** [comment]
@dgkanatsios The deploy.json is in your repo and I'm getting the same error.  But, the file exists at that link.  Any idea why it's failing?

**Comment by dgkanatsios** [comment]
weird. @ShinraCorpMixer can you try copying/pasting the template to an Azure Template Deployment? Just to make sure something hasn't changed in the ARM interpreter

**Comment by ShinraCorpMixer** [comment]
Hello @dgkanatsios I've tried copying the deploy.json file, however it brought up an error during validation:

"{"telemetryId":"89cbfe79-ad63-44bf-9a10-8c3731c49bc5","bladeInstanceId":"Blade_127527ee6cf1496eae622faf3ea8394e_0_0","galleryItemId":"Microsoft.Template","createBlade":"DeployToAzure","code":"InvalidTemplateDeployment","message":"The template deployment 'Microsoft.Template' is not valid according to the validation procedure. The tracking id is '1aa823b2-ab5a-47a4-bc20-4d34021d1ec4'. See inner errors for details.","details":[{"code":"ValidationForResourceFailed","message":"Validation failed for a resource. Check 'Error.Details[0]' for more information.","details":[{"code":"ServerFarmNotFound","message":"The specified app service plan was not found."}]}]}"

**Comment by ShinraCorpMixer** [comment]
Small update, I've forked the repo and modified the template URIs to point it to the forked one, seems to be working fine except for the error that I've posted earlier. I'm scratching my head as to why it says the the app service plan was not found. I'm still a novice when it comes to ARM templates so I'm not sure why it's not finding the App Service information even if there's a dependsOn for it at the bottom of the deploy.function.json file.

 "details": [
        {
          "code": "ServerFarmNotFound",
          "message": "The specified app service plan was not found."
        }
